snakebot:
    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 20

    joint_1_position_controller:
        type: effort_controllers/JointPositionController
        joint: joint_1
        pid: {p: 100.0, i: 30.0, d: 40.0}

    joint_2_position_controller:
        type: effort_controllers/JointPositionController
        joint: joint_2
        pid: {p: 100.0, i: 30.0, d: 40.0}

    joint_3_position_controller:
        type: effort_controllers/JointPositionController
        joint: joint_3
        pid: {p: 100.0, i: 30.0, d: 40.0}

    joint_4_position_controller:
        type: effort_controllers/JointPositionController
        joint: joint_4
        pid: {p: 100.0, i: 30.0, d: 40.0}

    joint_5_position_controller:
        type: effort_controllers/JointPositionController
        joint: joint_5
        pid: {p: 100.0, i: 30.0, d: 40.0}

    joint_6_position_controller:
        type: effort_controllers/JointPositionController
        joint: joint_6
        pid: {p: 100.0, i: 30.0, d: 40.0}

    joint_7_position_controller:
        type: effort_controllers/JointPositionController
        joint: joint_7
        pid: {p: 100.0, i: 30.0, d: 40.0}
    

